File: solutions/c/multiples_3_and_5.c
#include <stdio.h>


int main(int argc, char *argv[]) {
    unsigned int sum = 0;
    for (int i=0;i<1000;i++) {
        if (i%3==0 || i%5==0) {
            sum += i;
        }
    }
    printf("%u", sum);
}

File: solutions/java/Multiples3And5.java
class Multiples3And5 {
    public static void main(String[] args) {
        long sum = 0;
        for (int i = 0;i < 1000;i++) {
            if (i % 3 == 0 || i % 5 == 0) {
                sum += i;
            }
        }
        System.out.println(sum);
    }
}